code/src/pages/FilmDetail.js

```diff
@@ -0,0 +1,88 @@
+import React, { useState, useEffect } from 'react';
+
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+
+import './FilmDetail.css' 
+import { NotFound } from './NotFound';
+
+export const FilmDetail = () => {
+    const { filmid } = useParams();
+
+    const [loading, setLoading] = useState(true);
+    const [FilmDetails, setFilmDetails] = useState({}); 
```

Comment:
Capitalised names are reserved for components only, state properties should rather be written lowercase